packages/esm-service-queues-app/src/hooks/useQueueStatuses.ts

```diff
@@ -0,0 +1,37 @@
+import { type FetchResponse, OpenmrsResource, openmrsFetch, restBaseUrl, useOpenmrsSWR } from '@openmrs/esm-framework';
+import { useSystemSetting } from './useSystemSetting';
+import { useMemo } from 'react';
+import type { Concept } from '../types';
+import useSWRImmutable from 'swr/immutable';
+
+function useQueueStatuses() {
+  const {
```

Comment:
Same comment here as above.  Each queue has an `allowedStatuses` property, so we can iterate over each queue, collect each `allowedStatus` in a Set, sort it by name, and return it.  Or we can add a dedicated endpoint.